### DISCUSSION TITLE
A filter on the home feed to toggle on/off followed hashtags

### DISCUSSION THREAD

**Discussion by keithfancher** [pixelfed/pixelfed#5274] [Ideas]
It's great to follow hashtags to discover new stuff, new artists, etc. But sometimes, as a slightly different use-case, I just want to see what's new *specifically* with the users I follow. There's no easy way to do this now (that I can find) other than manually clicking through each user's feed.

I think it would be really handy to have some kind of toggle/filter on the "Home feed" to be able to switch on/off followed hashtags. (Or equivalent functionality, however it might be implemented.)

**Comment by jduar** [comment]
I'd love to see this suggestion implemented in some way. Here's my current issue:

Let's say I follow a few photographers whose work I really enjoy. But being interested in photography, I also follow a few popular photography hashtags. Given the chronological nature of the main feed, and how popular these hashtags are (fortunately), they completely drown out the content from the accounts I follow. Unless I happen to hop on Pixelfed at the exact same time one of my followed accounts happens to post, I'll never realistically scroll enough to reach their posts. Which means in practice following popular hashtags erases the possibility of seeing content from accounts I follow.

I think a possible solution would be to have a new page, sort of like discover, but only populated with followed hashtags. It could even be a tab within the discovery section for favourite hashtags.

Currently, the way I managed to achieve some balance was by unfollowing every hashtag and settings bookmarks for the hashtag specific pages I want to look at. It works but it isn't really a good user experience.
